packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -256,7 +256,14 @@ class AnalyzeCommand extends FlutterCommand {
       mainFile.path
     ];
 
-    printStatus(cmd.join(' '));
+    if (argResults['congratulate']) {
```

Comment:
I don't think we should overload `--congratulate` to do this. It would introduce additional lines of output when there are messages -- you just want the output to be the messages. We should introduce a new argument that does this, and default it to on.